integration_tests/e2e/bookAVisit.cy.ts

```diff
@@ -110,10 +122,10 @@ context('Book a visit', () => {
     const selectVisitorsPage = Page.verifyOnPage(SelectVisitorsPage)
     selectVisitorsPage.getPrisonerRestrictionType(1).contains(offenderRestrictions[0].restrictionTypeDescription)
     selectVisitorsPage.getPrisonerRestrictionComment(1).contains(offenderRestrictions[0].comment)
-    selectVisitorsPage
-      .getPrisonerRestrictionStartDate(1)
-      .contains(format(new Date(offenderRestrictions[0].startDate), mediumDateFormat))
-    selectVisitorsPage.getPrisonerRestrictionEndDate(1).contains('Not entered')
+    selectVisitorsPage.getPrisonerRestrictionEndDate(1).contains('No end date')
+    selectVisitorsPage.getPrisonerAlertType(1).contains('COVID unit management')
+    selectVisitorsPage.getPrisonerAlertComment(1).contains('Alert comment')
```

Comment:
need to add to one of the Cypress tests a test involving a multi-line comment to demonstrate the show/hide JS behaviour working. Happy to do this if you want.